node/runtime/src/lib.rs

```diff
@@ -701,6 +706,22 @@ impl_runtime_apis! {
 		}
 	}
 
+	impl transaction_payment_rpc_runtime_api::TransactionPaymentApi<
```

Comment:
Shall we have a macro to implement stuff like this? It feels pretty repetitive to implement all these forwarding traits. Like:
```rust
runtime_api!{
  impl TransactionPaymentApi<Block, AccountId, Balance, Call, SignedExtra> for Runtime {
    fn query_info_unsigned(AccountId, Call, SignedExtra) -> RuntimeDispatchInfo<Balance> delegate to TransactionPayment;
  }
}
```

Reply:
Only when I started writing it I realised that there is probably no way to make it any shorter, but leaving here for posterity, maybe someone will figure out how to do it better.

Reply:
Yes I agree and I still have it on my list to do this very soon; but since this is rather a small api and having it in soon for kusama would make many validators + Jaco happy, I decided to do that first. Noted and hopefully very soon we'll further discuss how this boilerplate can be put into a macro.